src/Business Foundation/App/NoSeries/src/Single/NoSeriesSequenceImpl.Codeunit.al

```diff
@@ -26,6 +26,11 @@ codeunit 307 "No. Series - Sequence Impl." implements "No. Series - Single"
 
     procedure GetNextNo(var NoSeriesLine: Record "No. Series Line"; UsageDate: Date; HideErrorsAndWarnings: Boolean): Code[20]
     begin
+        if (not NoSeriesLine.IsTemporary()) and (NoSeriesLine."Last Date Used" <> UsageDate) then begin
```

Comment:
maybe add a short comment why we have this like "the No. Series line is updated whenever the date changes"